lib/rake/install.rake

```diff
@@ -44,15 +44,16 @@ namespace :install do
         RakeUtils.bundle_install
         RakeUtils.python_venv_install
 
-        puts CDO.dashboard_db_writer
         if ENV['CI']
           if ENV['CI_JOB'] == 'unit_tests'
             # Prepare for dashboard unit tests to run.
             RakeUtils.rake 'db:create db:test:prepare'
           elsif ENV['CI_JOB'] == 'ui_tests'
-            # Start preparing for ui tests to run by creating database and tables.
-            # Seeding will be performed in a later step.
-            RakeUtils.rake 'db:create db:schema:load'
+            # If we don't already have a database that's been restored from a
+            # cache, prepare one from scratch by creating the database and loading
+            # the schema. If we do, make sure it's up to date with the branch
+            # being tested. Either way, seeding will be performed in a later step.
```

Comment:
This comment feels excessively verbose, but it might just be an indication that this whole `if ENV['CI']` block is excessively fragile. Excessive, but contextually appropriate.

What do y'all think?